_posts/blog/2019-09-19-2019-09-19-lab-meeting.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: post
+title: Lab Meeting on 19 September 2019 ### REPLACE `DATE` with the date, eg: 18 July 2019
+categories: blog
+excerpt: Notes from the group's weekly lab meeting ### ADD a short description (or keep that one if you'd)
+tags: [lab-meeting] ### INSERT TAGS IF APPLICABLE
+image:
+  feature:
+link:
+date: 2019-09-19 ### UPDATE date
+modified:
+share: true
+author: kirstie_whitaker ### CHANGE to your author name (in _data/authors.yml)
+---
+
+We didn't have a lab meeting today because the timing of the meeting coincided with the Turing's [Data Science for Mental Health Interest Group](https://turing-ds4mh.github.io/index.html).
+
+The two talks were excellent.
+[Neil Oxtoby](http://neiloxtoby.com/) presented an introduction to data-driven disease progression modelling, and [Alexandra Young](https://ayoungresearch.wordpress.com/) extended Neil's presentation with a specific example of data-driven modelling and progression of subtypes of dementia.
+
+This weekend - Saturday 21 September - is [World Alzheimer's Day](https://www.alzheimers.org.uk/get-involved/world-alzheimers-day) and while we waited we watched Samuel L Jackson explain how you - and an orange 🍊 - can eliminate the stigma and dispel the myth that there's nothing that can be done about dementia.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/L3AyD4RABOc" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
As on the other PR, this video doesn't render in Markdown but Jekyll may handle it for publishing.

Reply:
Thanks @sgibson91 - looks ok when rendered but I appreciate you checking